.github/workflows/create-github-release.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         uses: softprops/action-gh-release@v1
         with: 
           files: target/Crucials-*.jar
-          prerelease: true
+          draft: true
```

Comment:
That was the original intention: gives some time for manual verification and approval before release is public.